### PR TITLE
more helper documentation and a tweak

### PR DESCRIPTION
**Pull request by doganmeh**
added the new *attr* param to the docs

### CODE REVIEW

**Comment by doganmeh** [comment]
I also realized the _make_xeditable_ helper has a mutable _extra_attrs_ argument. I was not sure if that was on purpose. Otherwise, something along these lines should work better:

```
def make_xeditable(instance=None, extra_attrs=None, *args, **kwargs):
    extra_attrs = extra_attrs or []
```

**Comment by doganmeh** [comment]
I created the commit for just in case.

**Comment by doganmeh** [comment]
that sounds better to me too

**Comment by doganmeh** [comment]
or it could be field_value vs field_rich_value/field_display_value.

field_value better identified what it is, also there is a field_name in the kwargs.

File: datatableview/helpers.py [hunk 1]
```diff
@@ -47,6 +47,12 @@ def keyed_helper(helper):
     a custom callback method that simply returns the value of the ``make_boolean_checkmark()``
     helper.
 
+    'attr' is shorthand for key=lambda o: getattr(o, attr), allowing simpler syntax like
```
Comment (michaeldjeffrey) [inline datatableview/helpers.py]: Please update this reference to `attr` to be consistent with the rest of the docs.

File: datatableview/helpers.py [hunk 2]
```diff
@@ -47,6 +47,12 @@ def keyed_helper(helper):
     a custom callback method that simply returns the value of the ``make_boolean_checkmark()``
     helper.
 
+    'attr' is shorthand for key=lambda o: getattr(o, attr), allowing simpler syntax like 
+    helpers.link_to_model(attr='relatedobject')
+    
+    dot notation might be used to traverse deeper relations, e.g.,
+    helpers.link_to_model(attr='relatedobject.related_to_relatedobject
```
Comment (michaeldjeffrey) [inline datatableview/helpers.py]: ')

File: datatableview/helpers.py (hunk 1 above)
Comment (doganmeh) [inline datatableview/helpers.py]: How is it referred to in the rest of the docs?

**Comment by doganmeh** [comment]
I just realized changing what goes into kwargs (e.g., default_value or rich_value) will possibly break people's custom helpers.

File: datatableview/helpers.py (hunk 1 above)
Comment (michaeldjeffrey) [inline datatableview/helpers.py]: function parameters are double back-ticked ``attr``

**Comment by doganmeh** [comment]
I see :)

File: datatableview/helpers.py (hunk 1 above)
Comment (doganmeh) [inline datatableview/helpers.py]: I see :)

Comment (doganmeh) [inline datatableview/helpers.py]: for correction, should I delete the commit, amend it, or just fix it in a brand new commit? is there a preferred way?

**Comment by doganmeh** [comment]
incorporated requested format changes in above commit

**Pull request closed without merging**